src/texts.js

```diff
@@ -23,6 +23,14 @@ var texts = {
       website: 'hubzilla.org',
       website_protocol: 'https',
       github: 'https://github.com/redmatrix/hubzilla'
+    },
+    ganggo: {
+      title: 'GangGo',
+      subtitle: 'A Decentralized Social Network',
+      description: 'Open source, decentralized and privacy oriented social network written in GoLang',
+      website: 'https://github.com/ganggo',
```

Comment:
This is silly, but I just realized that this needs a little tunage. `website` should be without protocol like the others so just `github.com/ganggo` - the protocol part is added separately. I don't remember the reason for this - possibly to display it without protocol and not have to remove protocol, which I know would be super hard ;)